test_conformance/device_partition/test_device_partition.cpp

```diff
@@ -332,11 +335,15 @@ int test_device_set(size_t deviceCount, size_t queueCount, cl_device_id *devices
         {
             if( expectedResults[ i ] != outputData[ i ] )
             {
-                log_error( "ERROR: Sample data did not verify for queue %d on device %ld (sample %d, expected %d, got %d)\n",
-                    q, q % deviceCount, (int)i, expectedResults[ i ], outputData[ i ] );
+                log_error("ERROR: Sample data did not verify for queue %d on "
+                          "device %zu (sample %d, expected %d, got %d)\n",
+                          q, q % deviceCount, (int)i, expectedResults[i],
+                          outputData[i]);
```

Comment:
Nitpick: I think we could remove the cast on `i` here?

```suggestion
                log_error("ERROR: Sample data did not verify for queue %d on "
                          "device %zu (sample %zu, expected %d, got %d)\n",
                          q, q % deviceCount, i, expectedResults[i],
                          outputData[i]);
```

Reply:
Thanks; the old code isn't using the wrong printf specifier for `i` here, which is why I didn't catch this as part of this effort.  But yes, we can get rid of the cast and use `%zu` too; I've updated the PR.